dept44-models/src/test/java/se/sundsvall/dept44/models/api/paging/PagingMetaDataTest.java

```diff
@@ -11,16 +9,20 @@
 import static org.hamcrest.CoreMatchers.allOf;
 import static org.hamcrest.MatcherAssert.assertThat;
 
+import org.junit.jupiter.api.Test;
+
 class PagingMetaDataTest {
 
 	@Test
 	void testBean() {
-		assertThat(PagingMetaData.class, allOf(
-			hasValidBeanConstructor(),
-			hasValidGettersAndSetters(),
-			hasValidBeanHashCode(),
-			hasValidBeanEquals(),
-			hasValidBeanToString()));
+		assertThat(
+				PagingMetaData.class,
+				allOf(
```

Comment:
Should we change the "formatter template" to allow the previous format?

It might be just me, but I don't think the "newline after paranthesis" always is the best and most reabable choice?